### PR TITLE
Add option to create minio clusters

### PR DESCRIPTION
**Pull request by SuperQ**
Add a cluster list that overrides the datadirs list, this allows for
local creation of data storage directories separate from a list of
cluster members.

### CODE REVIEW

**Comment by SuperQ** [comment]
I don't know what's up with the test:
```
ERROR: Idempotence test failed because of the following tasks:
* [minio-fedora-27] => ansible-minio : enable and start the minio service
* [minio-ubuntu-16.04] => ansible-minio : enable and start the minio service
* [minio-centos-7] => ansible-minio : enable and start the minio service
* [minio-ubuntu-18.04] => ansible-minio : enable and start the minio service
* [minio-debian-9] => ansible-minio : enable and start the minio service
```

**Comment by atosatto** [comment]
If you do molecule converge and you connect to the containers is minio running?

**Comment by SuperQ** [comment]
Figured it out, Minio is very sensitive to the setup of volumes in the erasure code mode.

File: README.md [hunk 1]
```diff
@@ -58,6 +58,25 @@ minio_server_make_datadirs: true
...
+Set a list of nodes to create a [distributed cluster](https://docs.minio.io/docs/distributed-minio-quickstart-guide).
+
+In this mode, ansible will create your server datadirs, but use this list for the server startup.
+
+Example:
+
+```yaml
+minio_server_datadirs:
+  - '/minio-data'
+minio_server_cluster_nodes:
```
Comment (paulfantom) [inline README.md]: It needs at least 4 nodes, let's not confuse users with 3 nodes in example.

File: README.md [hunk 2]
```diff
@@ -58,6 +58,25 @@ minio_server_make_datadirs: true
...
+
+Set a list of nodes to create a [distributed cluster](https://docs.minio.io/docs/distributed-minio-quickstart-guide).
+
+In this mode, ansible will create your server datadirs, but use this list for the server startup.
+
+Example:
+
+```yaml
+minio_server_datadirs:
+  - '/minio-data'
```
Comment (paulfantom) [inline README.md]: This needs at least 4 directories, not 1.

**Comment by paulfantom** [comment]
Overall it looks good, I just dropped some comments regarding documentation.

**Comment by SuperQ** [comment]
@paulfantom I added some additional notes to the README.

File: README.md (hunk 2 above)
Comment (SuperQ) [reply README.md]: You need 4 storage targets over the whole cluster, not per server.

Comment (paulfantom) [reply README.md]: You're right.

**Pull request merged**